iothub/device/src/Transport/Http/HttpTransportHandler.cs

```diff
@@ -365,7 +92,7 @@ await _httpClientHelper
             _ = GetInvokeDeviceMethodOperationTimeout(methodInvokeRequest);
```

Comment:
This appears to be the only place ```GetInvokeDeviceMethodOperationTimeout``` is called, and its result is immediately discarded. It should be safe to remove this method

Reply:
Unresolving this because I still see ```GetInvokeDeviceMethodOperationTimeout``` in ```HttpTransportHandler```